account/accounts_geth.go

```diff
@@ -0,0 +1,42 @@
+// +build !nimbus
+
+package account
+
+import (
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/status-im/status-go/account/generator"
+)
+
+// GethManager represents account manager interface.
+type GethManager struct {
+	Manager
```

Comment:
```suggestion
	*Manager
```
would make more sense I think. If `GethManager` was used as a value, it could produce some strange results (`Manager` will be copied).

Reply:
Good point!